src/Framework/PaymentGateways/EventHandlers/DonationCancelled.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Give\Framework\PaymentGateways\EventHandlers;
+
+use Exception;
+use Give\Donations\Repositories\DonationRepository;
+use Give\Donations\ValueObjects\DonationStatus;
+use Give\Framework\PaymentGateways\Actions\UpdateDonationStatus;
+
+/**
+ * @unreleased
+ */
+class DonationCancelled
+{
+    /**
+     * @unreleased
+     *
+     * @throws Exception
+     */
+    public function __invoke(string $gatewayTransactionId, string $message = '', $skipRecurringDonations = false)
+    {
+        $donation = give(DonationRepository::class)->getByGatewayTransactionId($gatewayTransactionId);
```

Comment:
It looks like some of the event handlers are using the DonationRepository class while others are using the service container name directly like `give()->donations` - I prefer the latter as its one less import - can we just tighten up that consistency please?

```suggestion
        $donation = give()->donations->getByGatewayTransactionId($gatewayTransactionId);
```

Reply:
Resolved in 57775d55735b3629354c76be8883a0e991637568